spec/api.test.js

```diff
@@ -0,0 +1,43 @@
+const data = require("../database_reviews/database.js");
+const request = require("supertest");
+const app = require("../server/server.js");
+
+beforeAll(async () => {
+  await data.query(
+    "CREATE TABLE reviews (id INT NOT NULL AUTO_INCREMENT, text VARCHAR(255) NOT NULL, star INT, verified TINYINT(1), name VARCHAR(255), date VARCHAR(255), product_id INT NOT NULL, PRIMARY KEY (id))"
+  );
+});
+
+beforeEach(async () => {
+  await data.query(
+    "LOAD DATA LOCAL INFILE 'testReview.csv' INTO TABLE reviews FIELDS TERMINATED BY ',' LINES TERMINATED BY ',' (text, star, verified, name, date, product_id);"
+  );
+});
```

Comment:
Is there a reason why you're using `beforeEach`, `beforeAll`, `afterEach` and `afterAll` when you only have one test? Can't you just use either `beforeEach` and `afterEach` OR `beforeAll` and `afterAll`?

Reply:
I plan to write more tests later, so it would be helpful to keep them for now